playground/frontend/lib/modules/editor/components/editor_textarea.dart

```diff
@@ -82,7 +86,9 @@ class _EditorTextAreaState extends State<EditorTextArea> {
       webSpaceFix: false,
     );
 
-    _setTextScrolling();
+    if (!widget.isEmbedded) {
```

Comment:
Maybe it's better to rename it to the `enableScrolling`, so we can use it in the future on other pages.

Reply:
+1

Reply:
Thank you, renamed this one.